src/Plugin/Field/FieldType/PurchaseFieldItem.php

```diff
@@ -31,6 +31,7 @@
  *   id = "apigee_purchase",
  *   label = @Translation("Purchase rate plan"),
  *   description = @Translation("Purchase rate plan computed item."),
+ *   category = @Translation("Apigee"),
```

Comment:
"Apigee" isn't really a translatable string, but maybe you need to pass it in as translatable.

Reply:
I'm pretty sure it isn't required.